src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -187,7 +187,7 @@ enum {
     UCT_IB_MLX5_MD_FLAG_CQE128_ZIP           = UCS_BIT(11),
     /* Device performance is optimized when RDMA_WRITE is not used */
     UCT_IB_MLX5_MD_FLAG_NO_RDMA_WR_OPTIMIZED = UCS_BIT(12),
-    /* Device supports indirect xgvmi MR */
+    /* Device supports indirect xgvmi MR. This flag is removed if xgvmi access command fails */
```

Comment:
line seems too long
pls check code style output

Reply:
AFAIU format code is ok with line too ling in case of comments

Reply:
yeah, it doesn't warn about this, but pls split it anyways
pls see other warnings in https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=64013&view=logs&j=cc064a77-22b5-56bf-ecc0-70b5fe764261&t=aedfd754-44a6-53e6-6843-8659d800fee2